x.payment_batching/payment_batching.md

```diff
@@ -0,0 +1,307 @@
+batching* to reduce transaction sizes and fees by about 75% in
+practical situations.
+
+As of February 2019, payment batching is used by multiple popular
+Bitcoin services (mainly exchanges), is available as a built-in feature
+of many wallets (including Bitcoin Core), and should be easy to
+implement in custom wallets and payment-sending solutions.  On the
+downside, use of the technique can lead to temporary unexpected behavior
+for the receivers of payments and may result in a reduction of privacy.
+
+## Transaction size per receiver
+
+A typical Bitcoin transaction using P2WPKH inputs and outputs contains
+one input from the spender of about 67 vbytes and two outputs of about
+31 vbytes each, one to the receiver and one as change back to the
+spender.  An additional 11 vbytes are used for transaction overhead
+(version, locktime, and other fields).
+
+![Best-case P2WPKH vbytes per payment](img/p2wpkh-batching-best-case.png)
+
+If we add just 4 more receivers, including an additional 31 vbyte output
+for each one of them, but otherwise keep the transaction the same, the
+total size of the transaction becomes 264 vbytes.  Whereas the previous
+transaction used all 140 vbytes to pay a single receiver, the batched
+transaction uses only about 53 vbytes per receiver---a bit over 60%
+savings per payment.
+
+Extrapolating this simple best-case situation, we see that the number of
+vbytes used per receiver asymptotically approaches the size of a single
+output.  This makes the maximum savings possible a bit over 75%.
+
+![Saving rates for best, typical, and tough cases of payment batching](img/p2wpkh-batching-cases-combined.png)
+
+Realistically, the more a transaction spends, the more likely it is to
+need additional inputs.  This doesn't prevent payment batching from
+being useful, although it does reduce its effectiveness.  For example,
+we can imagine the tough case of a gambling site that receives bets of
+10 mBTC and pays out winnings of 100 mBTC, requiring at least 10 inputs
+for every output added.  Here, the maximum savings from payment batching
+(without optimizations described later) peak at only about 5%.
+
+Falling between these extremes are a large number of services who
+receive payments of about the same value as the payments they make, so
+for every output they add, they need to add one input on average.
+Savings in this typical case peak at about 30%.
+
+Services that find themselves frequently using more than one input per
+transaction may be able to increase their savings using a two-step
+procedure.  In the first step, multiple small inputs are
+[consolidated][chapter consolidation] into a single larger input using
+slow (but low-feerate) transactions that spend the service's money back
+to itself.  In the second step, the service spends from one of its
+consolidated inputs using payment batching and achieves the best-case
+efficiency described above.
+
+If we assume that consolidation transactions will pay only 20% of the
```

Comment:
I could probably generate one, but Statoshi's record of fee estimates is a good one and supports the conclusion (estimatesmartfee on my node says waiting 144 blocks is 1/10th the cost of waiting 6 blocks).  Instead of linking it here, however, I think perhaps we should just put the relevant information in the chapter about consolidations and then link to that.